### PR TITLE
Feature 385/setup directory api

### PR DESCRIPTION
**Pull request by Bagurp**
* `build.gradle` -> Add imports for DirectoryAPI, caching
* `FileServicesImpl.java` -> rename GoogleDriveAccessor to GoogleAccessor
* Rewrite and move `GoogleAuthenticator.java` and `GoogleDriveAccessor.java` to `/utils`
* `Application.yml` -> Create cache called "photo-cache", add scopes for apis
* `GoogleApiAccess.java` -> Create objects of `Directory` and `Drive` on server startup
* Endpoint for fetching PNG file (user image), tests. URL = `/services/member-profile/member-photo/work_email`
* Update development build to include gsuite-super-admin env variable

### CODE REVIEW

File: server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package com.objectcomputing.checkins.services.memberprofile;
 
+import com.objectcomputing.checkins.services.memberprofile.memberphoto.MemberPhotoService;
```
Comment (mkimberlin) [inline server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java:3]: No longer used...

Comment (Bagurp) [reply server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileController.java]: updated

File: server/src/test/java/com/objectcomputing/checkins/security/GoogleServiceConfigurationTest.java [hunk 2]
```diff
@@ -30,14 +30,16 @@ void testInstantiation() {
         assertNull(googleServiceConfiguration.getToken_uri());
```
Comment (meyerinj) [inline server/src/test/java/com/objectcomputing/checkins/security/GoogleServiceConfigurationTest.java:30]: A lot of tests are failing for me with `Message: Could not resolve placeholder ${DIRECTORY_ID}`. I think you have this set up locally but need to set it up in the test environment.

**Review by meyerinj — CHANGES_REQUESTED** [review]

File: server/src/test/java/com/objectcomputing/checkins/security/GoogleServiceConfigurationTest.java (hunk 2 above)
Comment (Bagurp) [reply server/src/test/java/com/objectcomputing/checkins/security/GoogleServiceConfigurationTest.java:30]: We've fixed this now. We do not need service account details for testing purpose. We've mocked all the calls that need this.

**Review by meyerinj — APPROVED** [review]

**Review by warnerj-oci — APPROVED** [review]

**Pull request merged**